exercises/concept.wip/documented-lasagna/.docs/hints.md

```diff
@@ -2,8 +2,6 @@
 
 ## General
 
-## 1. Document the three methods
-
 - Review the [Documentation][doc] section in the Julia Manual.
 - Start with the function signature indented with 4 spaces.
 - Include a single one-line sentence describing what the function does, in your own words.
```

Comment:
Delete this whole file?

Reply:
I think you can have general hints, but if your idea is to put the content in the instructions file is there no purpose of having a hints file I would say,